src/main/java/org/cbioportal/web/util/DataBinner.java

```diff
@@ -565,6 +647,16 @@ public Long countNAs(List<Binnable> clinicalData, ClinicalDataType clinicalDataT
         return count;
     }
 
+    /**
+     * Calculate number of clinical data marked actually as "NA", "NAN", or "N/A"
+     */
+    public Long countNAs(List<Binnable> clinicalData) {
```

Comment:
This is cast to Long and then later it grabs just the intValue(). Do you need the wrapper class?

        bin.setCount(countNAs(clinicalData).intValue());

Reply:
Probably we don't need a wrapper here. For now, just keeping it consistent with the legacy `countNAs` method.

https://github.com/cBioPortal/cbioportal/blob/de88ff79ad7bed99b706759e9ab0eff8e7d0fe1d/src/main/java/org/cbioportal/web/util/DataBinner.java#L621

We can revisit this once we completely get rid of the legacy functionality.